src/components/dialogs/parameters/sensi/sensitivity-analysis-parameters.js

```diff
@@ -544,7 +557,48 @@ export const SensitivityAnalysisParameters = ({
         resetSensitivityParametersAndProvider,
     ]);
 
-    const isMaxReached = () => Math.abs(analysisComputeComplexity) > numberMax;
+    const renderComputingEventLoading = () => {
+        return (
+            <Box sx={styles.modificationsTitle}>
+                <Box sx={styles.icon}>
+                    <CircularProgress
+                        size={'1em'}
+                        sx={styles.circularProgress}
+                    />
+                </Box>
+                <Typography noWrap>
+                    <FormattedMessage id={'computing in progress'} />
+                </Typography>
+            </Box>
+        );
+    };
+
+    const renderComputingEvent = () => {
+        return (
+            <Alert severity={isMaxReached ? 'error' : 'info'}>
+                {analysisComputeComplexity > 999999 ? (
```

Comment:
Why 999999? It should be 500 000 No?

Reply:
To limit the display of simulated calculations to 6 digits, if the number exceeds 999999
we display the message 'SimulatedCalculationExceedsLimit' with the aim of maintaining the box with a  fixed size."